apps/happy-blocks/index.php

```diff
@@ -97,13 +97,45 @@ function a8c_happyblocks_view_assets() {
 add_action( 'wp_enqueue_scripts', 'a8c_happyblocks_view_assets' );
 
 /**
- * Get the topic's selected help site's domain.
+ * Get the domain to use in the Pricing Plans block.
  *
- * @return string
+ * @return string The domain
  */
-function a8c_happyblocks_get_topic_domain() {
-	$topic_id = bbp_get_topic_id();
-	return get_post_meta( $topic_id, 'which_blog_domain', true );
+function a8c_happyblocks_pricing_plans_get_domain() {
+
+	// If the user is not authenticated, then we can't get their domain.
+	if ( ! is_user_logged_in() ) {
+		return null;
+	}
+
+	$topic_id  = bbp_get_topic_id();
```

Comment:
I can see it was there before but just noticed this: should we be concerned whether bbPress is loaded here? Like do a quick `class_exists( 'bbpress' )` or something before calling bbPress functions?